src/main/java/core/mate/academy/service/MachineService.java

```diff
@@ -16,7 +17,7 @@
      * @param type - any class of Machine sub class. For example: Truck.class or Bulldozer.class
      * @return the list of machines
      */
-    List<Object> getAll(Class type);
+    List<Machine> getAll(Class<? extends Machine> type);
```

Comment:
```suggestion
    List<T> getAll(Class<? extends T> type);
```
краще тут T використати, щоб код був гнучкішим, на випадок якщо потрібно створити TruckService який буде більш обмеженим